types/redux-form/lib/reduxForm.d.ts

```diff
@@ -15,15 +15,18 @@ import {
     WarningOther,
     RegisteredFieldState,
     FormStateMap
-} from "redux-form";
+} from "../index";
 
 export type FormSubmitHandler<FormData = {}, P = {}> =
     (values: Partial<FormData>, dispatch: Dispatch<any>, props: P) => void | FormErrors<FormData> | Promise<any>;
 
-export interface SubmitHandler<FormData = {}, P = {}> {
-    (handler: FormSubmitHandler<FormData, P>): FormEventHandler<any>;
-    (event: SyntheticEvent<any>): void;
-}
+export type SubmitHandler<FormData = {}, P = {}> = (
```

Comment:
Why this lane has been changed? This makes my code doesn't compile at all 😠 
![image](https://user-images.githubusercontent.com/10618781/34566453-f1b0db8e-f15e-11e7-8c19-42e4c78fa600.png)

Reply:
@19majkel94 feel free to send a new PR and we will merge it in

Reply:
@19majkel94 Hi! Fixed in #22945, could you please review it?